src/evolve_pressure.cxx

```diff
@@ -148,6 +148,16 @@ EvolvePressure::EvolvePressure(std::string name, Options& alloptions, Solver* so
   neumann_boundary_average_z = p_options["neumann_boundary_average_z"]
     .doc("Apply neumann boundary with Z average?")
     .withDefault<bool>(false);
+
+  numerical_viscous_heating = options["numerical_viscous_heating"]
+    .doc("Include heating due to numerical viscosity?")
+    .withDefault<bool>(false);
+
+  if (numerical_viscous_heating) {
+    fix_momentum_boundary_flux = options["fix_momentum_boundary_flux"]
+      .doc("Fix Y boundary momentum flux to boundary midpoint value?")
+      .withDefault<bool>(false);
```

Comment:
@bendudson how come `fix_momentum_boundary_flux` is only available with `numerical_viscous_heating`? I only just noticed this. 

Second question - are there any downsides to having numerical viscous heating enabled? At the moment it defaults to disabled, which would lead to a small energy imbalance.

Reply:
It's there because in `evolve_pressure`, `fix_momentum_boundary_flux` is only relevant if `numerical_viscous_heating` is enabled, because you need the operator to calculate it.